### PR TITLE
Upgrade to OSP 1.11

### PR DESCRIPTION
**Pull request by Roming22**
We can clean-up the commits if the tests are successful.

For chains:
    - configured chains to use the old namespace, so that we don't have to worry about migrating the signing key for the moment.
    - put the tekton-chains config in TektonConfig
    - no longer use the upstream release payload, and allow the operator to install chains
 For PaC, remove all PaC artifacts and let the operator install

### CODE REVIEW

**Comment by Roming22** [comment]
Upgrade test fails on `tests-post-upgrade` with:
```
[chains]
  - Run pipeline: Error: Pipeline name simple-copy does not exist in namespace plnsvc-tests
 ```

File: operator/images/access-setup/content/bin/setup_work_dir.sh [hunk 1]
```diff
@@ -157,8 +157,8 @@ tekton_chains_manifest(){
       --workdir /workspace \
       --entrypoint /usr/bin/cosign \
       "$cosign_image" generate-key-pair
-    kubectl create namespace tekton-chains --dry-run=client -o yaml > "$chains_namespace"
-    kubectl create secret generic -n tekton-chains signing-secrets --from-file="$chains_tmp_dir" --dry-run=client -o yaml | \
+    kubectl create namespace openshift-pipelines --dry-run=client -o yaml > "$chains_namespace"
```
Comment (gabemontero) [inline operator/images/access-setup/content/bin/setup_work_dir.sh:160]: why did you change the namespace here @Roming22 if in the tekton config you've switched the namespace to tekton-chains

File: operator/images/access-setup/content/bin/setup_work_dir.sh [hunk 2]
```diff
@@ -157,8 +157,8 @@ tekton_chains_manifest(){
...
+    kubectl create secret generic -n openshift-pipelines signing-secrets --from-file="$chains_tmp_dir" --dry-run=client -o yaml | \
```
Comment (gabemontero) [inline operator/images/access-setup/content/bin/setup_work_dir.sh:161]: same namespace question

File: operator/images/access-setup/content/bin/setup_work_dir.sh (hunk 1 above)
Comment (Roming22) [reply operator/images/access-setup/content/bin/setup_work_dir.sh]: That's me missing the change when doing the rollback. Fixed.

File: operator/images/access-setup/content/bin/setup_work_dir.sh (hunk 2 above)
Comment (Roming22) [reply operator/images/access-setup/content/bin/setup_work_dir.sh]: Same answer ;). Fixed.

**Comment by Roming22** [comment]
@gabemontero pipeline-service-test is now fixed. pipeline-service-upgrade-test is green until the rollback. AFAIU this is expected because there's no rolling back an operator upgrade.

I believe this change can be merged and deployed. I leave it up to you to either do it on your own next week, or if you're not comfortable rolling out the change while I'm away, we can do it the week after.

**Comment by gabemontero** [comment]
> @gabemontero pipeline-service-test is now fixed. pipeline-service-upgrade-test is green until the rollback. AFAIU this is expected because there's no rolling back an operator upgrade.
> 
> I believe this change can be merged and deployed. I leave it up to you to either do it on your own next week, or if you're not comfortable rolling out the change while I'm away, we can do it the week after.

OK we've had some passing e2e runs after rebasing my PRs 

And I also obtained a sanity check with the EC team over in https://redhat-internal.slack.com/archives/C02CTEB3MMF/p1689008673134839

My plan is to get https://github.com/redhat-appstudio/infra-deployments/pull/2083 merged first, so we can close out some grafana related jiras, and then I'll merge this and clean up my 2 POC PRs in infra and e2e-tests.

**Pull request merged**